arrows/ffmpeg/ffmpeg_video_input.cxx

```diff
@@ -131,26 +127,22 @@ class ffmpeg_video_input::priv
   // --------------------------------------------------------------------------
   priv() : m_klv_demuxer( m_klv_timeline ) {}
 
-  // ==================================================================
+  // --------------------------------------------------------------------------
 
```

Comment:
Any chance the scripts could remove blank lines like this? (IOW, between Doxygen and a separator line?)